lua/pendulum/init.lua

```diff
@@ -10,6 +10,13 @@ local default_opts = {
     timer_len = 120,
     gen_reports = true,
     top_n = 5,
+    sections = {
```

Comment:
I would rename this to make it a little bit more clear what the option is for
```suggestion
    report_sections = {
```

Reply:
that makes sense

Reply:
I changed `sections` to `group` under `report_excludes`. Hope that's also clear enough.